### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
Added homepage with JSX

### CODE REVIEW

File: src/components/Home-Banner/Homebanner.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Homebanner.css";
+
+function Homebanner({ imageUrl, heading, content, reversed }) {
+  return (
```
Comment (JowelTisso) [inline src/components/Home-Banner/Homebanner.jsx:6]: Try using camelCase for two different word

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          heading="Best of English Web Series"
+          content="Enjoy best hollywood web series seanlessly."
+          reversed="false"
+        />
+        <Homebanner
+          imageUrl={banner2}
+          heading="Best of Indian Web Series"
+          content="Enjoy best of indian web series on th go."
+          reversed="true"
+        />
```
Comment (JowelTisso) [inline src/pages/Home/Home.jsx:48]: You can create an array of data and map through it instead of manually placing the components

**Review by JowelTisso — APPROVED** [review]
* Try keeping initial PR and feature branch PR separate  
 * And try keeping PR as small as possible

**Review by ramalingaa — APPROVED** [review]
Code looks clean , good folder structure. PR can be merged

**Pull request merged**